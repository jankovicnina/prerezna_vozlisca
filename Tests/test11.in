49 68
5 37 0.72
13 24 -1.53
3 45 4.6
47 41 8.44
4 10 1.53
31 25 9.63
7 18 7.6
25 15 7.29
0 29 1.99
26 40 3.03
4 36 8.29
10 27 6.04
24 40 10.0
18 39 9.57
9 8 9.08
23 6 2.94
17 0 6.83
16 23 7.1
30 2 0.54
40 16 2.94
45 28 2.61
24 32 0.73
41 30 -1.54
1 31 7.25
6 22 4.98
8 43 -1.14
29 32 0.68
26 11 -1.27
20 14 2.12
21 42 6.18
27 13 -0.4
22 9 7.36
34 5 -0.27
0 30 9.98
33 32 3.99
3 17 0.0
24 1 9.1
38 47 9.85
11 20 9.84
35 11 -1.61
37 7 1.05
14 40 6.31
12 4 3.1
42 48 7.48
45 12 4.07
48 38 6.3
23 12 4.29
39 44 2.77
25 42 2.52
39 10 1.9
3 21 4.3
19 3 1.64
46 34 2.98
43 33 6.79
41 42 7.89
23 33 1.64
35 44 1.49
15 6 8.64
36 35 5.93
2 46 -1.07
28 21 7.47
33 17 6.43
24 10 4.72
32 26 2.53
47 6 8.7
44 19 1.12
38 19 7.27
10 6 -1.78
