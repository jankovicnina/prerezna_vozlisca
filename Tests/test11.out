1101 1264 1350 2048 3400 3686 4898 4943
